src/components/ErrorPage.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Container } from 'reactstrap';
+import { FormattedMessage } from 'react-intl';
+import { Link } from 'react-router-dom';
+
+import apiClient from '../config/apiClient';
+
+export default class ErrorPage extends Component {
+  componentDidMount() {}
+
+  renderProfileLink(linkText) {
+    const { username } = apiClient.getAuthenticationState().authentication;
+    return <Link to={`/u/${username}`}>{linkText}</Link>;
```

Comment:
I don't see an a11y-related things added by HyperLink in the paragon example. Also, I don't think HyperLink works with react-router. I think we are good with what we have here.

Reply:
Sorry - I just meant something like setting `aria-label`, since the text `here` doesn't really help.  And I see now that Hyperlink is only setting this for external links.